docs/api/com-atproto-server-delete-session.api.mdx

```diff
@@ -40,7 +40,7 @@ import Heading from "@theme/Heading";
 
 *To learn more about calling atproto API endpoints like this one, see the [API Hosts and Auth](/docs/advanced-guides/api-directory) guide.*
 
-Delete the current session. Requires auth.
+Delete the current session. Requires auth using the 'refreshJwt' token (not the 'accessJwt' token).
```

Comment:
Does it mean the request would be

```bash
curl --request POST \
  --url https://bsky.social/xrpc/com.atproto.server.deleteSession \
  --header 'Authorization: Bearer <refresh_token>' \
```
??

If so, I got `200 OK` response and then did `GET https://bsky.social/xrpc/com.atproto.server.refreshSession` with the refresh token and got 
```400 Bad Request
{
	"error": "ExpiredToken",
	"message": "Token has been revoked"
}
```

So it works, I confirm.